#!/usr/bin/env python
PACKAGE = "roomac"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("wait_time_before_picking_action", double_t, 0, "How long to wait for stabilization of artag position before picking is started", 5.0, 0, 20)

gen.add("artag_stable_position_threshold", double_t, 0, "How close filtered and real artag transform has to be to each other for detected position to be considered stable", 0.04, 0, 0.2)
gen.add("artag_stable_orientation_threshold", double_t, 0, "How close filtered and real artag transform has to be to each other for detected position to be considered stable (in degrees)", 5.0, 0.0, 90.0)

exit(gen.generate(PACKAGE, "roomac", "PickAndBringManagerSettings"))